lib/rspec/core/option_parser.rb

```diff
@@ -282,38 +283,64 @@ def configure_only_failures(options)
       add_tag_filter(options, :inclusion_filter, :last_run_status, 'failed')
     end
 
-    def initialize_project_and_exit
-      RSpec::Support.require_rspec_core "project_initializer"
-      ProjectInitializer.new.run
-      exit
-    end
+    module Callables
+      class << self
```

Comment:
I think I'd prefer if this just used `def self.` for each method or used `module_function`.